github-checks-app/README.md

```diff
@@ -0,0 +1,16 @@
+# [BluBracket](https://blubracket.com/) GitHub Checks App tools
+
+## Creating a GitHub Enterprise Server App
+1. Run `python github-checks-app-on-enterprise-server.py`
+2. When prompted, input your enterprise server domain
+3. Copy/paste the URL into a browser of your choice, scroll down to click `Create GitHub App`
+4. Take note of the `webhook_secret=<webhook_secret>` in the URL
+    ```
+    > python creating-github-enterprise-server-app.py
+    Enter your GitHub Enterprise Server domain name (i.e. github.companyname.com): github.blucodon.com
+    https://github.blucodon.com...webhook_secret=Z1KPOYifctpzOjfphKj_hqRlZbrDOBG9AU7hgj7iPrk...
```

Comment:
Is any portion of this secret part of an actual/real secret?

There is no reason to use github.blucodon.com as an example right? it only exposes more of our infrastructure? why not use acme.com?